bench.py

```diff
@@ -83,10 +84,17 @@ def calculator(
 @click.option('--vmprof',
     is_flag=True,
     help="Profling benchmarkers with Vmprof and upload the result to the web")
+@click.option(
+    '--set',
+    '-s',
+    multiple=True,
+    help="Settings to be passed to the Scrapy command. Use with the bookworm/broadworm commands.")
+
+
 @click.pass_context
-def cli(ctx, n_runs, only_result, upload_result, book_url, vmprof):
+def cli(ctx, n_runs, only_result, upload_result, book_url, vmprof, set):
```

Comment:
The extended name for the upstream `-s` option is `--set` (https://github.com/scrapy/scrapy/blob/1.7.4/scrapy/commands/__init__.py#L70-L71).
I generally don't like overriding built-ins, but I think there is no harm in this since it's only a one-line function.

Reply:
Makes sense, thanks for elaborating on this. I didn't know about the extended name.